### PR TITLE
amend about kubectl create secret of changelog

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:
kubectl create secret has a new option --from-env-file that populates a configmap from file which follows a key=val format for each line.
maybe copy the above line ,need change the configmap to secret,thanks!
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2071,7 +2071,7 @@ Features for this release were tracked via the use of the [kubernetes/features](
 
 		* `kubectl create configmap` has a new option `--from-env-file` that populates a configmap from file which follows a key=val format for each line. ([#38882](https://github.com/kubernetes/kubernetes/pull/38882), [@fraenkel](https://github.com/fraenkel))
```
Comment (xiangpengzhao) [inline CHANGELOG.md]: Since you are here, please also delete the extra indent.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2071,7 +2071,7 @@ Features for this release were tracked via the use of the [kubernetes/features](
...
 
-		* `kubectl create secret` has a new option `--from-env-file` that populates a configmap from file which follows a key=val format for each line.
+		* `kubectl create secret` has a new option `--from-env-file` that populates a secret from file which follows a key=val format for each line.
```
Comment (xiangpengzhao) [inline CHANGELOG.md]: and the indent here.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4440,8 +4440,8 @@ filename | sha256 hash
...
 * Be able to specify the timeout to wait for pod for kubectl logs/attach ([#41813](https://github.com/kubernetes/kubernetes/pull/41813), [@shiywang](https://github.com/shiywang))
 * Add support for bring-your-own ip address for Services on Azure ([#42034](https://github.com/kubernetes/kubernetes/pull/42034), [@brendandburns](https://github.com/brendandburns))
-* 1. create configmap has a new option --from-env-file that populates a configmap from file which follows a key=val format for each line. ([#38882](https://github.com/kubernetes/kubernetes/pull/38882), [@fraenkel](https://github.com/fraenkel))
-    * 2. create secret has a new option --from-env-file that populates a configmap from file which follows a key=val format for each line.
+* kubectl create configmap has a new option --from-env-file that populates a configmap from file which follows a key=val format for each line. ([#38882](https://github.com/kubernetes/kubernetes/pull/38882), [@fraenkel](https://github.com/fraenkel))
+    * kubectl create secret has a new option --from-env-file that populates a secret from file which follows a key=val format for each line.
```
Comment (xiangpengzhao) [inline CHANGELOG.md]: and here.

**Review by xiangpengzhao — CHANGES_REQUESTED** [review]
some nits.

**Comment by jianglingxia** [comment]
/assign @brendandburns 
for approve

**Comment by jianglingxia** [comment]
PTAL @smarterclayton thanks very much!
just now add some nit and squash into one commit!

**Comment by jianglingxia** [comment]
PTAL @smarterclayton  @thockin

**Pull request merged**